Dockerfile

```diff
@@ -91,7 +91,7 @@ USER neuro
 
 WORKDIR /src/nibetaseries
 
-RUN pip install --no-cache-dir .
+RUN conda run -n neuro_py36 pip install --no-cache-dir -e .
```

Comment:
`conda run` is currently my surefire way to ensure items are installed in the correct environment, making it explicit which pip I want to use.